data/human/campaign events.txt

```diff
@@ -21,22 +21,26 @@ mission "remembrance day"
 	source
 		government "Republic"
 		attributes "spaceport"
+	substitutions
+		"<first paragraph>" ""
+		"<first paragraph>" `	"`
+			has "non-human space start"
+		"<end paragraph>" ""
+		"<end paragraph>" `"`
+			has "non-human space start"
 	to offer
-		or
-			and
-				has "event: remembrance day"
-				not "hai start"
-			and
-				day > 19
-				month == 2
+		has "event: remembrance day"
 	on offer
 		log "Factions" "Alphas" `The Alphas were designed hundreds of years ago as genetically engineered super-soldiers, with no compassion or empathy. They escaped from their masters, enslaved entire planets, and commanded massive fleets of ships. Eventually they were defeated, and were either killed or went into hiding. They can be recognized by their green, photosynthetic skin.`
 			`Alphas can live for centuries, and it is likely that many of them have survived, hiding out on anarchist worlds or somewhere beyond human space. Fortunately, they reproduce much more slowly than ordinary humans do.`
 		conversation
-			`Today, throughout human space, is Remembrance Day, the anniversary of the beginning of the Alpha Wars more than five centuries ago. The Alphas were genetically engineered soldiers, with superhuman agility and intelligence, and a complete lack of compassion or empathy. At the height of the War, dozens of planets had been enslaved by the Alphas.`
+			`As you land on this human world, you receive an informative news messages titled 'Remembrance Day', noted as very important. It reads:`
```

Comment:
```suggestion
			`As you land on this human world, you receive an informative news bulletin titled 'Remembrance Day', noted as very important. It reads:`
```
Should be singular, but you can go with `message` if preferable. I prefer `bulletin`, personally.